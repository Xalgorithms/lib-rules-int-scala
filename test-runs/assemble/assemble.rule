REQUIRE test:origin:0.0.0 AS table_origin;
REQUIRE test:join:0.0.0 AS table_join;

ASSEMBLE table0
  COLUMNS FROM table:table_origin;

ASSEMBLE table1
  COLUMNS (a, c) FROM table:table_origin
  COLUMN p FROM table:table_join;