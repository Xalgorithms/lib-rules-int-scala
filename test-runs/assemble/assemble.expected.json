{
  "tables" : {
    "table" : {
      "table0" : [
        { "a" : 1, "b": 2, "c" : 3 },
        { "a" : 11, "b": 22, "c" : 33 }
      ],
      "table1" : [
        { "a" : 1, "c" : 3, "p" : "p0" },
        { "a" : 1, "c" : 3, "p" : "p1" },
        { "a" : 11, "c" : 33, "p" : "p0" },
        { "a" : 11, "c" : 33, "p" : "p1" }
      ]
    }
  }
}
